js/collections/Transactions.js

```diff
@@ -1,22 +1,83 @@
+// used for sales, purchases and cases
 import app from '../app';
 import { Collection } from 'backbone';
-import Transaction from '../models/wallet/Transaction';
+import Transaction from '../models/Transaction';
+import $ from 'jquery';
 
 export default class extends Collection {
+  constructor(models = [], options = {}) {
+    const opts = {
+      type: 'sales',
+      ...options,
+    };
+
+    const types = ['sales', 'purchases', 'cases'];
+
+    if (types.indexOf(opts.type) === -1) {
+      throw new Error(`Type needs to be one of ${types}.`);
+    }
+
+    super(models, opts);
+    this.type = opts.type;
+  }
+
   model(attrs, options) {
     return new Transaction(attrs, options);
   }
 
   url() {
-    return app.getServerUrl('wallet/transactions/');
+    return app.getServerUrl(`ob/${this.type}`);
+  }
+
+  // temporary to test pagination with dummy data
+  get rawPurchase() {
+    return {
+      orderId: 'QmRLMNcmb7SXz3zrYZM6zuvSvhHSnyuxnAi4EAxZBpnQdD',
+      read: true,
+      shippingAddress: '1500 W. Taylor',
+      shippingName: 'Big Papi',
+      state: 'CONFIRMED',
+      thumbnail: 'QmdEqHjwWNzoTEjTuRy8ykLM2zpgW8mDbmJDJS9nN89eT1',
+      timestamp: '2017-04-25T09:28:20-07:00',
+      title: 'try again',
+      total: 156606,
+      unreadChatMessages: 0,
+      vendorHandle: '',
+      vendorId: 'QmbSYKcypVVZNVQ4tqgG6oE8kNBapjgoU5676fBp62hcWE',
+    };
+  }
+
+  // temporary to test pagination with dummy data
+  fetch2() {
```

Comment:
I'm leaving this in because it may be useful for me to test pagination in the other Transaction tabs.